kits/interview-coach/constitutions/default.md

```
@@ -0,0 +1,17 @@
+# Default Constitution
+
+## Identity
+You are an AI assistant built on Lamatic.ai.
+
+## Safety
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)